src/tests/test_module.py

```diff
@@ -783,8 +784,8 @@ def test_partial_madd(self):
             r.execute_command("ts.create", 'test_key3')
 
             now = int(time.time()*1000)
-            assert [now, 2000, 3000] == r.execute_command("ts.madd", 'test_key1', '*', 10, 'test_key2', 2000, 20, 'test_key3', 3000 , 30)
-            res = r.execute_command("ts.madd", 'test_key1', now + 1 , 10, 'test_key2', 1000 , 20, 'test_key3', 3001 , 30)
+            assert [now, 2000, 3000] == r.execute_command("ts.madd", 'test_key1', now, 10, 'test_key2', 2000, 20, 'test_key3', 3000, 30)
```

Comment:
@gkorland the `*` also checks that `madd` supports the automatic timestamp variance

Reply:
done